cg/cli/upload/vogue.py

```diff
@@ -150,6 +157,41 @@ def bioinfo(context, case_name, cleanup, target_load, dry):
             context.obj["vogue_upload_api"].load_bioinfo_sample(load_bioinfo_raw_inputs)
 
 
+@vogue.command("bioinfo-all", short_help="Load all mip bioinfo results into vogue")
+@click.option("--dry/--no-dry", default=False, help="Dry run...")
+@click.pass_context
+def bioinfo_all(context, dry):
+    """Load all cases with recent analysis and a multiqc-json to the trending database."""
+
+    hk_api = context.obj["housekeeper_api"]
+    cases = context.obj["db"].families()
+    for case in cases:
+        case_name = case.internal_id
+        version_obj = hk_api.last_version(case_name)
+        if not version_obj:
+            continue
+
+        # confirm multiqc.json exists
+        multiqc_file_obj = hk_api.get_files(
+            bundle=case_name, tags=["multiqc-json"], version=version_obj.id
+        )
+        if len(list(multiqc_file_obj)) == 0:
+            continue
+
+        # confirm that file exists
+        existing_multiqc_file = multiqc_file_obj[0].full_path
+        if not Path(existing_multiqc_file).exists():
+            continue
+
+        click.echo(
+            click.style(f"Found multiqc for {case_name}, {existing_multiqc_file}", fg="blue")
+        )
+        try:
+            context.invoke(bioinfo, case_name=case_name, cleanup=True, target_load="all", dry=dry)
+        except AnalysisUploadError:
+            LOG.error("Case upload failed: %s", case_name, exc_info=True)
+
+
 def _get_multiqc_latest_file(context, case_name):
```

Comment:
Two things here 

- we don't need to send the whole context as we only need the housekeeper api
- Lets stick to python type hints as this is used everywhere in CG

I'll add a suggestion 
```suggestion
def _get_multiqc_latest_file(hk_api: HousekeeperAPI, case_name: str) -> str:
```

Reply:
I made some changes according to you comment. There is one left, which is: `get_links` and it seems it needs a click.Context: https://github.com/Clinical-Genomics/cg/blob/6577794299c6ae61a81057540ed83f1d2dcc246d/cg/cli/workflow/get_links.py#L10 I kept it as it is for this one.

Reply:
Ok, that one could be fixed by raising exceptions instead of `context.abort`. Otherwise it only uses `store` from the context. Feel free to adress that if you like :)